docs/release-notes/2.3.0.md

```diff
@@ -1,3 +1,7 @@
 
+## New features
+- Add a validation that forbids setting up secure and unsecure handlers on the same path [#915](https://github.com/kyma-project/api-gateway/pull/915)
```

Comment:
Should we "sell it" as a feature? I think it's more like a bugfix for bad validation. But maybe that's just me ... 😁